### ISSUE TITLE
Not all required JS assets are being loaded in 4.0.0

### ISSUE THREAD

**Issue by creativesuspects** [UmbHost/Our.Umbraco.Forms.uCaptcha#5]
I've upgraded from 3.1.2 to 4.0.0 and I'm using an inline hCaptcha. After the upgrade the local umbracoforms.hcaptcha.js resource containing the `onSubmit(token)` method is not being loaded anymore.  I'm using `@Html.RenderUmbracoFormDependencies()` to load the Umbraco Forms dependencies. Only https://hcaptcha.com/1/api.js is being loaded.

It looks like you may have removed the code that includes the local resource in this commit: https://github.com/UmbHost/Our.Umbraco.Forms.uCaptcha/commit/69415d920d35b206c4307eade82b6ffb00b02fb3

**Comment by AaronSadlerUK** [comment]
Have you got this in your `Partials/Forms/Themes/THEMENAME/Render.cshtml` file:

```
@if (Model.CurrentPage.JavascriptCommands.Any())
{
    <script>
    document.addEventListener("DOMContentLoaded", function() {
        @foreach (var javascriptCommand in Model.CurrentPage.JavascriptCommands)
        {
            @Html.Raw(javascriptCommand)
        }
    });
</script>
}
```

**Comment by creativesuspects** [comment]
Nope. I probably should've mentioned that I'm using Umbraco Forms 10.2.3 on Umbraco 10.4.0. And I've downloaded the Forms 10 Default Theme from the docs page: https://docs.umbraco.com/umbraco-forms/developer/themes to do some customization. However, the code you mentioned is not part of that default theme.

**Comment by AaronSadlerUK** [comment]
I've raised the issue with Umbraco here:

https://github.com/umbraco/Umbraco.Forms.Issues/issues/978

You will need to create a theme (which is best practice anyway) and add the above function, I have no idea why they have removed it from the core

**Comment by AaronSadlerUK** [comment]
Hmmm strange, it's that function which I override to load the required files:

https://github.com/UmbHost/Our.Umbraco.Forms.uCaptcha/blob/main/src/UmbracoForms/uCaptchaField.cs#L78

I'm on Discord (same name) if you want to work through it

**Comment by AaronSadlerUK** [comment]
Fixed in release #4.0.3
